packages/modules/pocket/src/pocket.ts

```diff
@@ -183,7 +183,13 @@ export class Pocket {
    */
   public withPrivateKey(privateKey: Buffer | string): ITransactionSender | Error {
     try {
-      const privKeyBuffer = typeGuard(privateKey, Buffer) ? privateKey  : Buffer.from(privateKey , 'hex')
+      let privKeyBuffer: Buffer | string
+
+      if (typeGuard(privateKey, Buffer)) {
+        privKeyBuffer = privateKey
+      }
+      privKeyBuffer = Buffer.from((privateKey as string), "hex")
```

Comment:
I think this is expected here, otherwise, even if `privateKey` is typeof Buffer it will try to convert it to buffer at line 191.
```suggestion
      if (typeGuard(privateKey, Buffer)) {
        privKeyBuffer = privateKey
      } else {
        privKeyBuffer = Buffer.from((privateKey as string), "hex")
      }
```

Reply:
For posterity, while I'm still not sure if it's possible to overhaul the `keybase` implementation so it doesn't need buffers (at least exposed in code), we should standardize on operating on these strings through a specific format (not having to convert things back and forth, but rather let the functions that need a specific format handle this conversion). While mostly a cosmetic thing, it does cause errors due to inconsistent APIs.